src/ComponentTester.cpp

```diff
@@ -26,20 +31,17 @@ void ComponentTester::testPin(){
     tone(pin, 1000, 100); //(Buzzer)_component.tone(1000) wouldn't work. Polymorphism?
   }
   if(componentType.equalsIgnoreCase("PHOTORESISTOR")){
-    for(int i = 1; i <= 10; i++){
+    /*for(int i = 1; i <= 10; i++){
```

Comment:
If we are just commenting this out, what purpose does it serve to have a test for the photoresistor?